lib/safe-pg-migrations/plugins/idem_potent_statements.rb

```diff
@@ -26,6 +26,12 @@ def remove_index(table_name, options = {})
       SafePgMigrations.say("/!\\ Index '#{index_name}' not found on table '#{table_name}'. Skipping statement.", true)
     end
 
+    def add_foreign_key(from_table, to_table, **options)
+      return super unless foreign_key_exists?(from_table, options.except(:validate).presence || to_table)
```

Comment:
I think we should slice `:column` and `:name` from options instead, as add_foreign_key has [other options](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/add_foreign_key) that can not be used to determine if the FK exists, ex: `add_foreign_key :comments, :users, on_delete: :cascade` will fail to check the presence of the FK

Reply:
@rchoquet I'll replace `options.except(:validate).presence || to_table` with `options.slice(: column, :name) || to_table`, it that okay for you?

Reply:
@rchoquet Hash.slice was introduced in ruby 2.5.0, should I bump the ruby version?
Or write a workaround?